tests/run-all.sh

```diff
@@ -1,6 +1,8 @@
 #! /bin/bash
 set -eu
 
+e2eRegex=${e2eTestRegex:-*.test.ts}
```

Comment:
```suggestion
E2E_REGEX=${E2E_TEST_REGEX:-*.test.ts}
```

I'd follow a convetion to use uppercase for bash variables.